samples/matter/smoke_co_alarm/README.rst

```diff
@@ -73,15 +73,38 @@ The sample supports ICD Dynamic SIT LIT switching (DSLS) feature to switch betwe
 The device uses the SIT mode, when the wired power source is active.
 Otherwise, it switches to the LIT mode, to indicate that it is possible to save the energy, when using a battery power source.
 Dynamic switching is possible only if the ICD device meets all criteria for operating in the LIT mode (it has at least one client registered).
-DSLS support is enabled by default.
-To disable it, set the :kconfig:option:`CONFIG_CHIP_ICD_DSLS_SUPPORT` Kconfig option to ``n``.
+DSLS support is disabled by default.
+To enable it, set the :kconfig:option:`CONFIG_CHIP_ICD_DSLS_SUPPORT` Kconfig option to ``y``.
 
 In the LIT mode, the device responsiveness is much lower than in the SIT mode.
 However, you can request the device to become responsive to, for example, change its configuration.
 To do that, you need to use the User Active Mode Trigger (UAT) feature by pressing the appropriate button.
 
 See `User interface`_ for information about how to switch the operation modes.
 
+System off configuration
+------------------------
+
+The sample supports optional system off configuration, that turns off the whole system once an ICD device enters the idle state.
+Turning off the system, instead of putting it to sleep allows the device to achieve much lower power consumption in the idle state.
+On the other hand, the device wakes up from a sleep state much faster than from the system off, and it sends only Thread data poll frame and Matter data report.
+In case of waking up from the system off, the device has to boot the whole system, initialize platform, re-attach to network and send data report every time, what consumes a lot of energy.
```

Comment:
```suggestion
In case of waking up from the system off, the device has to boot the whole system, initialize platform, re-attach to network, and send data report every time, which consumes a lot of energy.
```